<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt;
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasklistWidget">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>167</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_changes_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>390</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;
background: #01bfff;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>save changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;
background: #01bfff;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>add new</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24pt;
background: #01bfff;
color:white;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>DAILY TASK PLANNER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_task_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>deafult text</string>
   </property>
  </widget>
  <widget class="QLabel" name="add_new_task_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:7pt;
</string>
   </property>
   <property name="text">
    <string>add new task here : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_new_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 11pt;
background: #01bfff;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>save new</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
